<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPhasePageLayerDialog</class>
 <widget class="QDialog" name="SelectPhasePageLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidgetPhase1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetPage1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelLayer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelect1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetLayer1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
